Fixes fake buttons on the push.bdnewsx.com

wout-codes opened the pull request:
Fake button and useless things.
Link: `https://push.bdnewsx.com/demo`

Here are the necessary screenshots https://github.com/AdguardTeam/AdguardFilters/pull/75108#issuecomment-781201007

wout-codes committed:
Update specific.txt

Fake button and useless things.
Link: `https://push.bdnewsx.com/demo`

zloyden commented:
> Fake button and useless things.
> Link: `https://push.bdnewsx.com/demo`

Unfortunately, I got immediately redirect to the `techrfour.com`. For such requests without task you should provide atleast a screenshot.

wout-codes commented:
> > Fake button and useless things.
> > Link: `https://push.bdnewsx.com/demo`
> 
> Unfortunately, I got immediately redirect to the `techrfour.com`. For such requests without task you should provide atleast a screenshot.

@zloyden Yep, they redirects always. By the way, there are placeholders in that site.

zloyden commented:
> By the way, there are placeholders in that site.

Fixed for `techrfour.com`. What about your rule `push.bdnewsx.com##.banner`?

wout-codes commented:
@zloyden Placeholders here: `https://techymozo.com/5-facts-about-mobile-apps/`

wout-codes commented:
@zloyden 
![image](https://user-images.githubusercontent.com/76908410/108324523-fae8e480-7202-11eb-9dfb-862e8b1ea7fb.png)

wout-codes committed:
Merge branch 'master' into patch-173

zloyden commented:
> I also added `techymozo.com` because of leftovers.

It was fixed. Please also use the spoiler for the screenshots.

wout-codes commented:
> > I also added `techymozo.com` because of leftovers.
> 
> It was fixed. Please also use the spoiler for the screenshots.

Ok.

wout-codes committed:
Update specific.txt

wout-codes commented:
@zloyden You can merge this

zloyden commented:
> You can merge this

Please read [this](https://github.com/AdguardTeam/AdguardFilters/pull/75108#issuecomment-781121129).

wout-codes commented:
@zloyden 
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/76908410/108332911-4b644000-720b-11eb-9cd5-1cd48e2ebcc3.png)
</details><br/>

zloyden commented:
So are we talking about `Click here to download`? Can you show the element with opened devtools?

wout-codes commented:
@zloyden 

<details><summary>Screenshot:</summary>

! With the rules I added: 
![image](https://user-images.githubusercontent.com/76908410/108334081-8dda4c80-720c-11eb-8ffb-60cc8357f33b.png)
</details><br/>

<details><summary>Screenshot:</summary>

! Without the rules I added:
![image](https://user-images.githubusercontent.com/76908410/108334181-a34f7680-720c-11eb-89f0-b8a2f029135e.png)
</details><br/>

<details><summary>Screenshot:</summary>

! DevTools:
![image](https://user-images.githubusercontent.com/76908410/108334306-c1b57200-720c-11eb-8910-999f34161989.png)
</details><br/>

zloyden commented:
Ok, thanks. I edited your first post by adding a link to the screenshots. Please do the same in the future.

wout-codes commented:
> Ok, thanks. I edited your first post by adding a link to the screenshots. Please do the same in the future.

Okay.

zloyden commented on EnglishFilter/sections/specific.txt:
```diff
@@ -224,7 +224,7 @@ xxxparodyhd.net,movgotv.com,123kmovies.net,123movies.*,bluemovies.co,f5movies.to
 ! common names
 ! ##.banner
-earncoin.site,rb7url.net,teenlolly.com,best18porn.com,fotodovana.com,1shorten.com,shortlinkpay.com,dextools.io,alternativeto.net,nakedneighbour.com,viral-topics.com,recipestutorials.com,tawiia.com,18tube.xxx,18porno.tv,youx.xxx,xz2.xyz,caixinglobal.com,bitcoinly.in,fakemail.net,internewstv.com,trinddz.com,omni.se,earnguap.com,forums.sonarr.tv,cashurl.in,abdeo8.com,xfantazy.com,printscreenshot.com ...
+earncoin.site,push.bdnewsx.com,rb7url.net,teenlolly.com,best18porn.com,fotodovana.com,1shorten.com,shortlinkpay.com,dextools.io,alternativeto.net,nakedneighbour.com,viral-topics.com,recipestutorials.com,tawiia.com,18tube.xxx,18porno.tv,youx.xxx,xz2.xyz,caixinglobal.com,bitcoinly.in,fakemail.net,internewstv.com,trinddz.com,omni.se,earnguap.com,forums.sonarr.tv,cashurl.in,abdeo8.com,xfantazy.com,pr ...
```
One last thing. `push.bdnewsx.com` should be at the beginning.

wout-codes replied:
Ok added.

wout-codes committed:
Update specific.txt
